package.json

```diff
@@ -11,7 +11,11 @@
     "test": "jest --ci --testResultsProcessor='jest-junit' --passWithNoTests --colors --coverage",
     "lint": "eslint . --ext .ts",
     "lint-and-fix": "eslint . --ext .ts --fix",
-    "doc": "typedoc --out public --readme README.md --exclude \"**/__tests__/**/*.+(ts|tsx|js)\" --exclude \"**/jest.config.js\" --excludeNotDocumented --name \"Paxful SDK (Javascript)\" --hideGenerator --disableSources --highlightTheme light-plus src/"
+    "gen": "npm run gen-paxful && npm run gen-webhook && npm run gen-fix-names",
```

Comment:
How are we going to manually publish schemas to this repository and build them as part of CI/CD?

Reply:
Currently, we will do it locally only. Full automation is not possible at the moment